src/spikeinterface/core/recording_tools.py

```diff
@@ -118,7 +118,7 @@ def get_closest_channels(recording, channel_ids=None, num_channels=None):
     return np.array(closest_channels_inds), np.array(dists)
 
 
-def get_noise_levels(recording, return_scaled=True, **random_chunk_kwargs):
+def get_noise_levels(recording, return_scaled: bool = True, method: str = "mad", **random_chunk_kwargs):
```

Comment:
Thanks for adding typings, could you add the rest?
```
def get_noise_levels(
    recording: BaseRecording, 
    return_scaled: bool = True, 
    method: Literal['std', 'mad'] = 'mad', 
    **random_chunk_kwargs
) -> np.array
```

Reply:
Ah didn't know about `Literal`, that's quite neat!
Will add it in the next commit :)